2019-11-13 18:58:03,124 : kivy : Factory: 184 symbols loaded
2019-11-13 18:58:03,127 : kivy : Cache: register <kv.lang> with limit=None, timeout=None
2019-11-13 18:58:03,200 : kivy : Cache: register <kv.image> with limit=None, timeout=60
2019-11-13 18:58:03,200 : kivy : Cache: register <kv.atlas> with limit=None, timeout=None
2019-11-13 18:58:03,216 : kivy : Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
2019-11-13 18:58:03,216 : kivy : Cache: register <kv.texture> with limit=1000, timeout=60
2019-11-13 18:58:03,216 : kivy : Cache: register <kv.shader> with limit=1000, timeout=3600
2019-11-13 18:58:03,267 : kivy : VideoGstplayer: Using Gstreamer 1.16.0.0
2019-11-13 18:58:03,267 : kivy : Video: Provider: gstplayer
2019-11-13 18:58:03,335 : kivy : Loading MPF-MC controller
2019-11-13 18:58:03,335 : mpfmc : Mission Pinball Framework Media Controller v0.53.0-dev.15
2019-11-13 18:58:03,335 : mpfmc : Mission Pinball Framework Game Engine v0.53.0-dev.67
2019-11-13 18:58:03,336 : mpfmc : Machine path: D:\mpf\wows\wows
2019-11-13 18:58:03,337 : ConfigProcessor : Config file in cache changed: D:\mpf\wows\wows\config\config
2019-11-13 18:58:03,337 : ConfigProcessor : Loading config from file c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml.
2019-11-13 18:58:03,337 : YAML File Interface : Loading file: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml
2019-11-13 18:58:03,381 : ConfigProcessor : Loading config: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml
2019-11-13 18:58:03,387 : ConfigProcessor : Loading config from file D:\mpf\wows\wows\config\config.
2019-11-13 18:58:03,387 : YAML File Interface : Loading file: D:\mpf\wows\wows\config\config.yaml
2019-11-13 18:58:03,407 : ConfigProcessor : Loading config: D:\mpf\wows\wows\config\config
2019-11-13 18:58:03,411 : kivy : Config File Error in ConfigProcessor: Found a "enable_events:" section in config file D:\mpf\wows\wows\config\config, but that section name is unknown. Did you mean "ball_devices:" instead?. Context: D:\mpf\wows\wows\config\config Error Code: CFE-ConfigProcessor-3 (https://docs.missionpinball.org/en/dev/logs/CFE-ConfigProcessor-3.html)
Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 146, in _check_sections
    if config_type not in config_spec[k]['__valid_in__']:
KeyError: 'enable_events'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\commands\mc.py", line 213, in __init__
    thread_stopper=thread_stopper).run()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 97, in __init__
    self.machine_config = self._load_config()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 223, in _load_config
    mpf_config = self.mpf_config_processor.load_config_files_with_cache(files, "machine", True)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 113, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 178, in _load_config_file_and_return_loaded_files
    self._check_sections(config, config_type, filename, ignore_unknown_sections)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 159, in _check_sections
    filename)
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigProcessor: Found a "enable_events:" section in config file D:\mpf\wows\wows\config\config, but that section name is unknown. Did you mean "ball_devices:" instead?. Context: D:\mpf\wows\wows\config\config Error Code: CFE-ConfigProcessor-3 (https://docs.missionpinball.org/en/dev/logs/CFE-ConfigProcessor-3.html)
2019-11-13 18:58:03,412 : kivy : Stopping child threads... (0 remaining)
2019-11-13 18:58:03,412 : kivy : All child threads stopped.
